dashboard/app/models/plc/enrollment_unit_assignment.rb

```diff
@@ -0,0 +1,47 @@
+# == Schema Information
```

Comment:
Represents the users enrollment in a specific course unit. Your course unit assignment sits on top of your module assignments - see the PLC object wiki for details